internal/website/content/tutorials/cli.md

```diff
@@ -0,0 +1,254 @@
+---
+title: "Tutorial: Command-Line Uploader"
+linkTitle: "Command-Line Uploader"
+date: 2019-03-19T18:45:53-07:00
+weight: 1
+---
+
+This quickstart will build a command line application called `upload` that
+uploads files to blob storage on both AWS and GCP. Blob storage stores binary
+data under a string key, and is one of the most frequently used cloud
+services.
+
+<!--more-->
+
+Before we start with any code, it helps to think about possible designs. We
+could very well write a code path for Amazon's Simple Storage Service (S3) and
+another code path for Google Cloud Storage (GCS). That would work. However, it
+would also be tedious. We would have to learn the semantics of uploading files
+to both blob storage services. Even worse, we would have two code paths that
+effectively do the same thing, but would have to be maintained separately
+(ugh!). It would be much nicer if we could write the upload logic once and
+reuse it across providers. That's exactly the kind of [separation of
+concerns][] that the Go CDK makes possible. So, let's write some code!
+
+When we're done, our command line application will work like this:
+
+```shell
+# uploads gopher.png to GCS
+$ ./upload gs://go-cloud-bucket gopher.png
+
+# uploads gopher.png to S3
+$ AWS_REGION=us-west-1 ./upload s3://go-cloud-bucket gopher.png
```

Comment:
Is this absolutely necessary to specify explicitly? I somehow can run simple samples without it

Reply:
Ah yes, @vangent obviated this in #1418. Good catch.